app/Http/Controllers/Api/V1/OrganisationController.php

```diff
@@ -1323,17 +1321,8 @@ public function getRegistries(Request $request, int $id): JsonResponse
                     'registry.affiliations.modelState.state',
                     'modelState.state'
                 ])
-                ->where(function ($query) use ($registryIds, $showPending, $id) {
-                    $query->whereIn('registry_id', $registryIds);
-
-                    if ($showPending) {
-                        $pendingInviteUserIds = PendingInvite::where([
-                            'organisation_id' => $id,
-                            'status' => config('speedi.invite_status.PENDING')
-                        ])->pluck('user_id');
-
-                        $query->orWhereIn('id', $pendingInviteUserIds);
```

Comment:
we're removing displaying pending invites? if we are on the FE, not still useful to keep in the BE?